src/hashtables.jl

```diff
@@ -11,40 +11,35 @@
 
     Fieldnames:
 
-    - maxOrder: maximum order (degree) of the polynomials
-    - numVars : maximum number of variables
+    - order:     order (degree) of the polynomials
+    - num_vars : number of variables
 
     These parameters can be changed using `set_params_TaylorN(order,numVars)`
     """ ->
 type ParamsTaylorN
-    maxOrder :: Int
-    numVars  :: Int
+    order :: Int
+    num_vars  :: Int
     variable_names :: Array{UTF8String,1}
 end
 
+global const _params_TaylorN_ = ParamsTaylorN(6, 2, UTF8String["x₁", "x₂"])
+
+
 @doc """Display the current parameters for `TaylorN` and
 `HomogeneousPolynomial`""" ->
 function show_params_TaylorN()
     info( """Parameters for `TaylorN` and `HomogeneousPolynomial`:
-    Maximum order       = $(_params_taylorN.maxOrder)
-    Number of variables = $(_params_taylorN.numVars)
-    Variable names      = $(_params_taylorN.variable_names)
+    Maximum order       = $(_params_TaylorN_.order)
```

Comment:
Is it reasonable to use here `get_order()`, `get_numvars()` and `get_variable_names()`?